airqo-platform/AirQo-frontend
# [NetManager] Full multi tenancy support

#### Summary of Changes (What does this PR do?)
- Add full multi-tenancy support

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Pull and checkout to this [branch](https://github.com/airqo-platform/AirQo-frontend/tree/ch-merge-configs-PLAT-117) locally
* Install node requirements `npm install`
* Start the application `npm run stage-mac` (on mac) or `npm run stage-pc` (on windows platform)
* You should be able to hit all the api services (in other words, the app should load all the data without any errors)
* More enphasis should be put on `auth-service` and `device registry` endpoints.

#### What are the relevant tickets?
- [PLAT-211](https://airqoteam.atlassian.net/browse/PLAT-211)

#### Screenshots (optional)

> Please add run command for Windows too.
